sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveCompatibilitySuite.scala

```diff
@@ -221,9 +221,6 @@ class HiveCompatibilitySuite extends HiveQueryFileTest with BeforeAndAfter {
     "udf_when",
     "udf_case",
 
-    // Needs constant object inspectors
-    "udf_round",
```

Comment:
Yes, I saw you still comment it out in whitelist, which mean we don't run `udf_round` in unit testing, so let's keep it in the blacklist, sound reasonable? Or can we fix this in this PR?